modules/oas-page-builder/src/services/database.ts

```diff
@@ -1,15 +1,15 @@
 import { Db, MongoClient } from 'mongodb';
 import { OASFile, OASFileGitHash } from './models/OASFile';
 
-const DB_NAME = process.env.DB_NAME || 'snooty_dotcomprd';
+const DB_NAME = 'snooty_dotcomprd';
```

Comment:
minor, dont forget to revert here

Reply:
This change is intentional. `DB_NAME` in the Autobuilder seems to be tied to either `pool` or `pool_test`, but we need a `snooty_*` database since this is where the OAS-related data is currently located. For the sake of simplicity, I think it makes sense to tie it down to the database and collection we actually need for now.

(Although in the future/long-term, we should probably move this OAS-related collection to be in `pool` or `pool_test`, or some other db entirely, as OpenAPI support evolves)

Reply:
wouldn't this result in test data getting into prod docs? also did not see the DB_NAME in env files, idk where `pool` would be coming from..

Reply:
`snooty_dotcomprd` is the same database that the frontend pulls OpenAPI data from to build the Cloud Admin API on the client-side. There shouldn't be any test data present in that database.

As for the `DB_NAME`, I think it was being declared/passed down from the [ECS container definitions](https://github.com/mongodb/docs-worker-pool/blob/8aab9e6afd765fff1af0e5743e3077434a0a4bad/infrastructure/ecs-main/ecs_service.yml#L113-L114).

Reply:
Just a heads up that I added the logic to use the Autobuilder's `SNOOTY_ENV` env var in order to allow us to access either `snooty_dotcomstg.oas_files` or `snooty_dotcomprd.oas_files`. 👍